src/core/pin.zig

```diff
@@ -13,18 +13,27 @@ const board = @import("board");
 pub fn Pin(comptime spec: []const u8) type {
     // TODO: Depened on board and chip here
 
-    // Pins can be namespaced with "Board:" for board and "Chip:" for chip
+    // Pins can be namespaced with "board:" for board and "chip:" for chip
+    // These namespaces are not passed to chip.parsePin
     const pin = if (std.mem.startsWith(u8, spec, "board:"))
-        chip.parsePin(@field(board.pin_map, spec))
+        chip.parsePin(@field(board.pin_map, spec[6..]))
```

Comment:
put "board:" and "chip:" into variables so that you can replace these magic numbers with something like `board_prefix.len`.